lib/Doctrine/Persistence/ObjectManager.php

```diff
@@ -111,9 +115,9 @@ public function flush();
     /**
      * Gets the repository for a class.
      *
-     * @param string $className
```

Comment:
I think this line should be preserved to allow IDE know about required argument type.

Reply:
I agree. Do you mind creating a pull request?

Reply:
Here it is #111